src/core/bonded_interactions/bonded_interaction_data.cpp

```diff
@@ -99,31 +60,28 @@ double maximal_cutoff_bonded() {
   return (any_dihedrals) ? 2 * max_cut_bonded : max_cut_bonded;
 }
 
+int bonded_ia_params_num_partners(int bond_id) {
```

Comment:
```suggestion
int no_of_partners(int bond_id) {
```

Reply:
The `bonded_ia_params_` prefix is used in this function and the ones above and below because it's acting on the variant called `bonded_ia_params`. This will become clear once we add the correct doxygen group delimiters. We could even move these functions to `src/script_interface`.

Reply:
Function moved to `src/script_interface/interactions/bonded.hpp` and modified to call a new function with signature `number_of_partners(Bonded_ia_parameters const &iaparams)` in `src/core/bonded_interactions/bonded_interaction_data.cpp`.